.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: python
+cache: pip
+
+matrix:
+  include:
+    - env: TOXENV=py27-dj18
+      python: 2.7
+    - env: TOXENV=py27-dj110
+      python: 2.7
+    - env: TOXENV=py35-dj18
```

Comment:
Good question. I have 3.5 via Homebrew. We can try to standardize against a Python 3 version - I know right now [wagtail-flags](https://github.com/cfpb/wagtail-flags/blob/master/tox.ini#L15) and  [wagtail-sharing](https://github.com/cfpb/wagtail-sharing/blob/master/tox.ini#L14) use Python 3.5, but that might be due to an arbitrary choice as opposed to a deliberate selection.